Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM ubuntu:20.04
+ARG IMAGE_VERSION="ubuntu:20.04"
```

Comment:
Why is this Dockerfile still needed? Seems to be mostly a copy of `Deploy.Dockerfile`.